torch/fx/graph.py

```diff
@@ -80,6 +80,8 @@ def graph_copy(self, g : 'Graph'):
         """
         val_map : Dict[Node, Node] = {}
         for node in g._nodes:
+            if node.op == 'output':
+                continue
```

Comment:
`graph_copy` is a bit unfinished: it probably should take val_map as argument, and then return the output node of the graph being copied. If that were so this would turn into a return.